source/imaer-gml/src/main/java/nl/overheid/aerius/gml/v5_0/togml/GMLVersionWriterV50.java

```diff
@@ -181,6 +181,19 @@ private CalculationMetadata getCalculation(final MetaDataInput input) {
     return calculation;
   }
 
+  private String convertToOldCalculationType(final MetaDataInput input) {
+    final CalculationMethod calculationMethod = input.getOptions().getCalculationMethod();
+
+    if (calculationMethod == null) {
+      return null;
+    }
+    if (calculationMethod == CalculationMethod.PERMIT) {
+      return CalculationMethod.PERMIT.type();
+    } else {
+      return calculationMethod.type();
+    }
```

Comment:
Not sure why this if/else is needed. Wouldn't just returning `calculationMethod.type()` do the same thing?

Also, I think we can get rid of the whole `v5_0/togml/` package now (perhaps different PR).